arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/CircuitBreaker.kt

```diff
@@ -2,8 +2,11 @@ package arrow.fx.coroutines
 
 import arrow.core.Either
 import arrow.core.identity
+import kotlin.time.Duration
```

Comment:
Breaking change

Reply:
Ok, I will create duplicated the descriptors with arrow.fx.Duration and create a PR for deprecating arrow.fx.coroutines.Duration